src/components/GuestList/GuestList.tsx

```diff
@@ -54,31 +93,32 @@ const GuestList: React.FC = () => {
           onChange={event => setGuestName(event.target.value)}
         />
         <input
-          type='tel'
-          placeholder='Phone Number'
+          type='text'
+          placeholder='Phone Number (XXX-XXX-XXXX)'
           name='phoneNumber'
-          maxLength={10}
-          value={phoneNumber !== 0 ? phoneNumber : ''}
-          onChange={event => setPhoneNumber(parseInt(event.target.value))}
+          maxLength={12}
+          value={phoneNumber}
+          onChange={event => checkNumber(event.target.value)}
         />
-        <StyledButton onClick={event => submitGuest(event)}>
+        <StyledButton>
+          <div id="translate"></div>
+          <a className="link" id="addListButton" onClick={event => submitGuest(event)}>Add To Guest List</a>
+        </StyledButton>
+        <StyledButton>
           <div id="translate"></div>
-            <a className="link" id="addListButton">Add To Guest List</a>
+          <a className="link" id="clearFormButton" onClick={event => clearInputs()}>Clear Guest and Phone Number</a>
```

Comment:
This works great for now! We'll keep looking for a solution, but this is a nice little bandaid for the time being